server.go

```diff
@@ -105,18 +112,19 @@ func NewExtensionManagerServer(name string, sockPath string, opts ...ServerOptio
 	}
 
 	manager := &ExtensionManagerServer{
-		name:         name,
-		sockPath:     sockPath,
-		registry:     registry,
-		timeout:      defaultTimeout,
-		pingInterval: defaultPingInterval,
+		name:                            name,
+		sockPath:                        sockPath,
+		registry:                        registry,
+		timeout:                         defaultTimeout,
+		pingInterval:                    defaultPingInterval,
+		serverConnectivityCheckInterval: time.Millisecond * 5, // Thrift's default value
```

Comment:
Can we leave it 0, and not pass it along if not set?

Reply:
I defaulted it to `math.MinInt64` since zero is a valid value, but avoid passing it along now if it's explicitly set to something else.